Signal/src/ViewControllers/HomeView/Chat List/ChatListViewController+Helpers.swift

```diff
@@ -145,8 +145,6 @@ public extension ChatListViewController {
             owsFailDebug("Missing threadViewModel.")
             return nil
         }
-        self.lastViewedThread = threadViewModel.threadRecord
```

Comment:
This didn't seem right. I'm not sure why we'd want to adjust our scroll offset based on our last thread that we displayed in a preview platter.

Reply:
Seems okay to me, given how we're using `lastViewedThread` elsewhere.